src/modules/python/pbs/v1/_svr_types.py

```diff
@@ -39,7 +39,7 @@
  */
 """
 
-__doc__ = r"""
+__doc__ = """
```

Comment:
There are 3 """ such quotes starting from line 40  to line 46. Shouldn't they be even in count.